WordPress/WordPressShareExtension/MainShareViewController.swift

```diff
@@ -82,8 +82,13 @@ private extension MainShareViewController {
 
     func loadAndPresentNavigationVC() {
         editorController.context = self.extensionContext
-        editorController.dismissalCompletionBlock = {
-            self.extensionContext?.completeRequest(returningItems: nil, completionHandler: nil)
+        editorController.dismissalCompletionBlock = { (exitSharing) in
+            if exitSharing {
+                self.extensionContext?.completeRequest(returningItems: nil, completionHandler: nil)
+            } else {
+                let error = NSError(domain: "com.wordpress.sharing", code: 0, userInfo: nil)
```

Comment:
Should we use a constant for the error code, with a descriptive name like ShareCancelled?

Reply:
Hmm... good question.  After a second look I've opted to just make an error enum with a single canceled case.